process-compose-mp.yml

```diff
@@ -0,0 +1,70 @@
+version: "3"
+
+environment:
+  - ESPRESSO_SEQUENCER_GENESIS_FILE=data/genesis/demo-marketplace.toml
+  - ESPRESSO_BUILDER_GENESIS_FILE=data/genesis/demo-marketplace.toml
+processes:
+  submit-transactions-public:
+    command: submit-transactions
+    environment:
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MIN_NAMESPACE=1
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MAX_NAMESPACE=6
+
+  submit-transactions-private:
+    command: submit-transactions
+    environment:
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MIN_NAMESPACE=1
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MAX_NAMESPACE=3
+
+  submit-transactions-private-fallback:
+    command: submit-transactions
+    environment:
+      - ESPRESSO_SUBMIT_TRANSACTIONS_PORT=$ESPRESSO_SUBMIT_TRANSACTIONS_PRIVATE_PORT
+      - ESPRESSO_SUBMIT_TRANSACTIONS_SUBMIT_URL=http://localhost:$ESPRESSO_FALLBACK_BUILDER_SERVER_PORT/txn_submit
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MIN_NAMESPACE=4
+      - ESPRESSO_SUBMIT_TRANSACTIONS_MAX_NAMESPACE=6
+    depends_on:
+      sequencer0:
+        condition: process_healthy
+      permissionless-builder:
+        condition: process_healthy
+    readiness_probe:
+      http_get:
+        scheme: http
+        host: localhost
+        port: $ESPRESSO_SUBMIT_TRANSACTIONS_PRIVATE_PORT
+        path: /healthcheck
+      failure_threshold: 100
+
+  submit-transactions-private-reserve:
+    command: submit-transactions
+    environment:
+      - ESPRESSO_SUBMIT_TRANSACTIONS_PORT=$ESPRESSO_SUBMIT_TRANSACTIONS_PRIVATE_PORT
+      - ESPRESSO_SUBMIT_TRANSACTIONS_SUBMIT_URL=http://localhost:$ESPRESSO_BUILDER_SERVER_PORT/txn_submit
```

Comment:
different port for reserve builder

Reply:
marketplace-reserve-builder has port set to `$ESPRESSO_BUILDER_SERVER_PORT`

Reply:
I thought we'd be running all three builders at once in marketplace native demo but I guess its fine to use legacy port for reserve one